telemetry_examples/src/main/java/com/newrelic/telemetry/examples/TelemetryClientExample.java

```diff
@@ -28,25 +32,40 @@
   public static void main(String[] args) throws Exception {
     String insightsInsertKey = args[0];
 
-    MetricBatchSender batchSender =
+    MetricBatchSender metricBatchSender =
         SimpleMetricBatchSender.builder(insightsInsertKey, Duration.of(10, ChronoUnit.SECONDS))
             .build();
     SpanBatchSender spanBatchSender = SimpleSpanBatchSender.builder(insightsInsertKey).build();
 
-    TelemetryClient telemetryClient = new TelemetryClient(batchSender, spanBatchSender);
+    // todo: make a simple version of this.
```

Comment:
let's create an issue for this, as well, so we don't lose track.

Reply:
#140 filed to track this.